### PR TITLE
GUI-2891: Network ACL details page WIP

### PR DESCRIPTION
**Pull request by kamalgill**
https://eucalyptus.atlassian.net/browse/GUI-2891

### CODE REVIEW

File: eucaconsole/static/js/pages/network_acl_view.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright 2017 Ent. Services Development Corporation LP
+ *
+ * @fileOverview Network ACL Details Page JS
+ * @requires AngularJS
+ *
+ */
+
+angular.module('NetworkACLDetailsPage', ['TagEditorModule'])
+    .controller('NetworkACLDetailsPageCtrl', function ($scope) {
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/network_acl_view.js]: Could simply define the module and not the controller if the intent is to define the dependency. I've done that elsewhere.

**Comment by dkavanagh** [comment]
retest this please

File: eucaconsole/static/js/pages/network_acl_view.js (hunk 1 above)
Comment (kamalgill) [inline eucaconsole/static/js/pages/network_acl_view.js]: There will probably be code here as the NACL details page is more fully implemented.  If not, I'll remove the controller part.

Comment (dkavanagh) [inline eucaconsole/static/js/pages/network_acl_view.js]: I was figuring the NACL could be handled like the tag editor, but guess see how that plays out.

**Pull request merged**